<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="newFilePushButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/misc_files/pictures/new-project.png</normaloff>:/misc_files/pictures/new-project.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openPushButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/misc_files/pictures/folder.png</normaloff>:/misc_files/pictures/folder.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="readOnlyPushButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/misc_files/pictures/read_only.png</normaloff>:/misc_files/pictures/read_only.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="savePushButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/misc_files/pictures/save.png</normaloff>:/misc_files/pictures/save.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="currentLanguageLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="aboutPushButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/misc_files/pictures/help.png</normaloff>:/misc_files/pictures/help.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="boldPushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="toolTip">
          <string extracomment="Bold"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="underlinePushButton">
         <property name="toolTip">
          <string extracomment="Underline text"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionRead_Only"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="geometry">
      <rect>
       <x>584</x>
       <y>101</y>
       <width>126</width>
       <height>96</height>
      </rect>
     </property>
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionRussian"/>
    </widget>
    <widget class="QMenu" name="menuTheme">
     <property name="title">
      <string>Theme</string>
     </property>
     <addaction name="actionDark"/>
     <addaction name="actionLight"/>
    </widget>
    <addaction name="menuLanguage"/>
    <addaction name="actionPreferences"/>
    <addaction name="menuTheme"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionRead_Only">
   <property name="text">
    <string>Read Only</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
